index.js

```diff
@@ -146,8 +165,43 @@ module.exports = function(app) {
         }
       ]
     }
-    app.signalk.addDelta(delta)
+    debug('Sending path zones notification:', JSON.stringify(notificationDelta))
+    app.signalk.addDelta(notificationDelta)
   }
 
+  function sendZonesMetaConfiguration(zones) {
+    const metaDelta = {
+      context: "vessels." + app.selfId,
+      updates:
+        zones.map(zone => {
```

Comment:
Even if I got the naming in the plugin schema wrong let's make things better here and rename the variable to something like `zonesEntry`. `zone.zones` a few lines below hurts my brain....

Reply:
Agreed! What about pathKey.zones.zone ?

Ideally we would rename the schema but that would invalidate the configuration. We could release under a new plugin name: Zones and leave to old one behind. Or maybe we do that when we move the configuration to baseDelta and handle all meta keys?

Reply:
Went to your suggestion...